src/main/java/com/pgs/booking/model/dto/CreateUpdateReservationDto.java

```diff
@@ -0,0 +1,19 @@
+package com.pgs.booking.model.dto;
+
+import com.pgs.booking.model.Reservation;
+import lombok.*;
+
+import java.util.List;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class CreateUpdateReservationDto {
+
+    private Reservation.ReservationStatus status;
+    private Long flight_id;
+    private List<CreateUpdatePassengerDto> passengers;
```

Comment:
if we require passengers (I guess yes)  set proper validation annotation for list

Reply:
changed and outdated.